be/src/column/object_column.cpp

```diff
@@ -251,6 +251,12 @@ void ObjectColumn<T>::put_mysql_row_buffer(starrocks::MysqlRowBuffer* buf, size_
     buf->push_null();
 }
 
+template <>
+void ObjectColumn<BitmapValue>::put_mysql_row_buffer(starrocks::MysqlRowBuffer* buf, size_t idx) const {
```

Comment:
This will change the default behavior, why do you change it? Could you share some background?

Reply:
I remember we changed it to NULL from the string before. Because when some bitmap is very big, it will cost much memory and it is not very convenient for user to execute `select * from table`
So when I see you change this behavior, I want to know the context of this change.

Reply:
Fine. I would close this PR. I want to change it because it's not straightforward to see a null value for bitmap